src/ui/EditTask.svelte

```diff
@@ -68,23 +68,45 @@
     const priorityOptions: {
             value: typeof editableTask.priority,
             label: string,
-            symbol: string }[] =
+            symbol: string,
+            accessKey: string,
+            accessKeyIndex: number}[] =
```

Comment:
There is something odd going on with the highlighting of the selected Priority options in the new code.

Previously, the whole word was marked in bold, for the selected priority:

<img width="491" alt="image" src="https://github.com/obsidian-tasks-group/obsidian-tasks/assets/4840096/71a654b4-1734-4c00-ab3a-713f858d5afc">


Now, the selected values look like this, where some words are not highlighted at all, and some are partially highlighted.

<img width="491" alt="image" src="https://github.com/obsidian-tasks-group/obsidian-tasks/assets/4840096/358a6bb7-e6d3-4132-9cf2-87db8d2325f5">

Reply:
yup, didn't see that, nice catch, fixed!

Reply:
Great - thank.